apps/react-workshop/src/Anchor.stories.tsx

```diff
@@ -20,3 +20,11 @@ export const AsButton = () => {
     </Anchor>
   );
 };
+
+export const External = () => {
+  return (
+    <Anchor href='https://www.example.com/' isExternal>
```

Comment:
nit: Since you also added `target="_blank"` in the docs example, maybe you can add it here also.

Reply:
Done [`ea66af0`](https://github.com/iTwin/iTwinUI/pull/1856/commits/ea66af0239cb1390f896235dbe461d384ac057c6)